src/base/bittorrent/session.cpp

```diff
@@ -62,10 +62,6 @@ using namespace BitTorrent;
 #include <libtorrent/extensions/smart_ban.hpp>
 //#include <libtorrent/extensions/metadata_transfer.hpp>
 
-#ifndef DISABLE_COUNTRIES_RESOLUTION
-#include "base/net/geoipmanager.h"
```

Comment:
I meant the first thing. I see that it isn't used anyway there that's why you remove it.